redpanda/values.yaml

```diff
@@ -566,3 +467,114 @@ config:
   # rpc_server_tls                                TLS configuration for RPC server                                validate
   # seed_servers                                List of the seed servers used to join current cluster; If the seed_server list is empty the node will be a cluster root and it will form a new cluster                                None
   # superusers                                List of superuser usernames                                None
+
+#
+# Listener overrides
```

Comment:
Is there a reason why this section was moved to the bottom? The reason it was closer to the top was because this is an important section that will likely need to be updated by the user based on their environment. With this change, this section is at the bottom past other sections that are both long and also not likely to need to be changed (and are mostly commented out). The original idea was that the sections would be primarily in order based on importance and then alphabetical.

Reply:
I moved it because it overrides generalized settings. Every setting above this has global consequences, the majority of the things that would be changed in the listeners section are overrides for things like tls enablement or external access.

Reply:
From a user perspective, I was going through enabling all the external access for everything so I could test it. After I got done doing that, I found the single global configuration I could have done it with.